### PR TITLE
AddingTo3.8_DataLakeLoading

### PR DESCRIPTION
**Pull request by JosephNewmanTG**
Associated task: https://graphsql.atlassian.net/browse/DOC-1885

Adding Data Lake Loading content to 3.8

### CODE REVIEW

File: modules/data-loading/pages/spark-connection-via-jdbc-driver.adoc [hunk 1]
```diff
@@ -20,10 +21,288 @@ If you use spark job to connect TigerGraph via JDBC, we recommend your concurren
...
-—num-executors 2  /* 2 executors per job */
-—executor-cores 2 /* 1 executor take 2 cores */
+—-num-executors 2  /* 2 executors per job */
+—-executor-cores 2 /* 1 executor take 2 cores */
 ----
+====
+
+== Load From a Data Lake Via Spark
+
+=== Prerequisites: General Loading from Spark
```
Comment (victorleeTG) [inline modules/data-loading/pages/spark-connection-via-jdbc-driver.adoc]: I think this section is not "Prerequisites" but the actual instructions for how to do basic Spark loading.
Also, the 5 bullets points should be "write a loading job" (1) which includes these 4 details.  So the hierarchy/structure of presenting that info might be done a little differently.

Comment (JosephNewmanTG) [reply modules/data-loading/pages/spark-connection-via-jdbc-driver.adoc]: Thank you these have been addressed with latest commit.

File: modules/data-loading/pages/spark-connection-via-jdbc-driver.adoc [hunk 2]
```diff
@@ -20,10 +21,303 @@ If you use spark job to connect TigerGraph via JDBC, we recommend your concurren
...
+—-num-executors 2  /* 2 executors per job */
+—-executor-cores 2 /* 1 executor take 2 cores */
 ----
+====
+
+== Load From a Data Lake via Spark
+. Use the `write()` function of the `DataFrame` to build a Spark `DataFrameWriter`.
++
+.. Specify the `mode("overwrite")` to set the save mode.
+.. Specify `format("jdbc")` to leverage the xref:https://spark.apache.org/docs/latest/sql-data-sources-jdbc.html[JDBC Data Source].
```
Comment (JosephNewmanTG) [inline modules/data-loading/pages/spark-connection-via-jdbc-driver.adoc]:
```suggestion
.. Specify `format("jdbc")` to leverage the https://spark.apache.org/docs/latest/sql-data-sources-jdbc.html[JDBC Data Source].
```

File: modules/data-loading/pages/spark-connection-via-jdbc-driver.adoc [hunk 3]
```diff
@@ -20,10 +21,303 @@ If you use spark job to connect TigerGraph via JDBC, we recommend your concurren
...
+.Example 4: Path to table:
+[source, gsql]
+val df = spark.read.format("delta")
+    .option("readChangeFeed", "true")
+    .option("startingTimestamp", "2021-04-21 05:45:46")
+    .load("pathToMyDeltaTable")
+
+. Filter the changes:
++
+The DataFrame containing the changes of the Delta table has xref:https://docs.databricks.com/en/delta/delta-change-data-feed.html#what-is-the-schema-for-the-change-data-feed[3 additional columns]:
```
Comment (JosephNewmanTG) [inline modules/data-loading/pages/spark-connection-via-jdbc-driver.adoc]:
```suggestion
The DataFrame containing the changes of the Delta table has https://docs.databricks.com/en/delta/delta-change-data-feed.html#what-is-the-schema-for-the-change-data-feed[3 additional columns]:
```

File: modules/data-loading/pages/spark-connection-via-jdbc-driver.adoc [hunk 4]
```diff
@@ -20,10 +21,303 @@ If you use spark job to connect TigerGraph via JDBC, we recommend your concurren
...
+====
+
+== Load From a Data Lake via Spark
+. Use the `write()` function of the `DataFrame` to build a Spark `DataFrameWriter`.
++
+.. Specify the `mode("overwrite")` to set the save mode.
+.. Specify `format("jdbc")` to leverage the https://spark.apache.org/docs/latest/sql-data-sources-jdbc.html[JDBC Data Source].
+.. Specify the JDBC connection properties in configuration options.
+. Create a GSQL xref:gsql-ref:ddl-and-loading:creating-a-loading-job.adoc[Loading Job], where you map the source column index to target vertex/edge attribute.
+
```
Comment (victorleeTG) [inline modules/data-loading/pages/spark-connection-via-jdbc-driver.adoc]:
```suggestion
```

File: modules/data-loading/pages/spark-connection-via-jdbc-driver.adoc [hunk 5]
```diff
@@ -20,10 +21,303 @@ If you use spark job to connect TigerGraph via JDBC, we recommend your concurren
...
 
 ----
-—num-executors 2  /* 2 executors per job */
-—executor-cores 2 /* 1 executor take 2 cores */
+—-num-executors 2  /* 2 executors per job */
+—-executor-cores 2 /* 1 executor take 2 cores */
 ----
+====
+
+== Load From a Data Lake via Spark
```
Comment (victorleeTG) [inline modules/data-loading/pages/spark-connection-via-jdbc-driver.adoc]:
```suggestion
== Load from a Data Lake via Spark
```

**Review by victorleeTG — CHANGES_REQUESTED** [review]

**Pull request merged**